docs/Projects/LLM-Benchmarks/index.md

```diff
@@ -0,0 +1,27 @@
+# LLM Benchmarks
+
+This document contains the design decisions behind the LLM Benchmarks the architectural diagrams and the technical
+solution can be found [here](https://github.com/MinBZK/llm-benchmark).
+
+The requirements of the solution are:
+
+- People should be able to run the LLM Benchmark system locally
```

Comment:
What does locally mean? On their machine or on a organizational resource?

Reply:
Within their internal company network, as some governmental deparment would have stricter network policies specifically for data

Reply:
To me, locally implies 'on my own machine'. To avoid confusion, I'd take Berry's comment and do something like:
```suggestion
- People should be able to run the LLM Benchmark system on the company's (internal) organizational resources
```